dashboard/charts.py

```diff
@@ -132,6 +132,21 @@ def get_waffle_chart(data: pd.DataFrame):
     return fig
 
 
+def get_waffle_metrics(data: pd.DataFrame):
```

Comment:
Great simple function to get the metrics so it's visually more understandable when looking at our waffle chart.